### PR TITLE
Persistent Message Saving - WIP

### PR DESCRIPTION
**Pull request by sargken**
This is a port from the system on the Altis Life Forums. If you find errors please tell me. 

P.S. Idk whats up with those commits below. I cant get rid of them.

### CODE REVIEW

File: AltisLife.sql [hunk 1]
```diff
@@ -166,8 +166,7 @@ CREATE TABLE IF NOT EXISTS `containers` (
 -- Table structure for table `wanted`
 -- Needed for extDB latest update on git
 --
-
-CREATE TABLE `wanted` (
+CREATE TABLE IF NOT EXISTS `wanted` (
```
Comment (sargken) [inline AltisLife.sql]: @danielstuart14 This is a bug i found

**Comment by tkcjesse** [comment]
To get rid of your previous commits you should just re-fork the repo, that is what I do at least when I want to add changes. That way you won't have your prior commits showing.

**Comment by Heavybob** [comment]
Thanks for the commit @sargken, will check it out and provide feedback where I can.

**Comment by Heavybob** [comment]
Going to need to check for conflicts with playtime update.

**Comment by sargken** [comment]
Im going to re make the pull with how @tkcjesse said.

**Pull request closed without merging**